environment/worker/src/main/resources/application.properties

```diff
@@ -0,0 +1,7 @@
+spring.application.name=worker
+logging.level.org.springframework=WARN
+spring.main.log-startup-info=false
+spring.main.banner-mode=OFF
+evaluator.consume.url=http://localhost:2137/consume
+evaluator.complete.url=http://localhost:2137/complete
```

Comment:
Sprawdzales moze jak to sie zachowa w docker swarmie/k8s? Bo ja szczerze mowiac nie wiem jakie url beda wystawione(tzn te chyba sa zle bo localhost w kontwnerze ma inny kontekst zupelnie)